src/Tribe/Tickets.php

```diff
@@ -1051,73 +1051,85 @@ public static function add_frontend_stock_data( array $tickets ) {
 		 */
 		public static function get_ticket_counts( $event_id ) {
 
-			$tickets = self::get_all_event_tickets( $event_id );
+		    	$tickets = self::get_all_event_tickets( $event_id );
```

Comment:
I hate to say it ... but throughout the change the initial indent is still out of whack. Right now we have `TAB TAB 4xSPACES TAB` but what we _want_ is `TAB TAB TAB` (`"draw_white_space": "all"` can help if you're using Sublime for example)